benchmarks/experiments/algos/ddpg_garage_tf.py

```diff
@@ -46,14 +46,16 @@ def ddpg_garage_tf(ctxt, env_id, seed):
     with LocalTFRunner(ctxt) as runner:
         env = TfEnv(normalize(gym.make(env_id)))
 
-        action_noise = OUStrategy(env.spec, sigma=hyper_parameters['sigma'])
-
         policy = ContinuousMLPPolicy(
             env_spec=env.spec,
             hidden_sizes=hyper_parameters['policy_hidden_sizes'],
             hidden_nonlinearity=tf.nn.relu,
             output_nonlinearity=tf.nn.tanh)
 
+        action_noise = OUStrategy(env.spec,
```

Comment:
continually the generality theme, maybe we should rename the algo param this is passed to `exploration_policy`?

Reply:
May as well.